src/smc-project/configuration.ts

```diff
@@ -63,12 +78,14 @@ async function sage_info(): Promise<{
   if (exists) {
     // We need the version of sage (--version runs quickly)
     try {
-      const info = (await exec("sage --version")).stdout.trim();
+      const info = (
+        await exec(`env PATH="${PATH}" sage --version`)
```

Comment:
Why would you do it this way, rather than using the env option to exec?   https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback

Reply:
hmm, I didn't think about that. Setting env var directly is better.

Reply:
I encourage you to do so in a followup PR.  I didn't make the change myself.